docs/source/coredev/index.rst

```diff
@@ -105,6 +105,13 @@ for the release you are actually making::
     VERSION=5.0.0
     BRANCH=master
 
+For `reproducibility of builds <https://reproducible-builds.org/specs/source-date-epoch/>`_,
+we recommend setting ``SOURCE_DATE_EPOCH`` prior to running the build; record the used value
+of ``SOURCE_DATE_EPOCH`` as it may not be available from build artifact. You
```

Comment:
This actually gives me an idea for flit; when SDE is set, it could record somewhere in the resulting file:

> Built with SOURCE_DATE_EPOCH=123 and flit version 0.12. See https://reproducible-builds.org/ for more info.

Reply:
I was thinking about that as well, in particular in IPython we have a  file that record the built commit; we could also record the timestamp.